celery/platforms.py

```diff
@@ -772,27 +773,72 @@ def ignore_errno(*errnos, **kwargs):
 
 
 def check_privileges(accept_content):
+    pickle_or_serialize = True if ('pickle' in accept_content or 'application/x-python-serialize' in accept_content) else False
```

Comment:
```suggestion
    pickle_or_serialize = 'pickle' in accept_content or 'application/x-python-serialize' in accept_content
```

This already evaluates to a boolean.